pandas/tests/indexing/test_iloc.py

```diff
@@ -118,6 +118,40 @@ def check(result, expected):
         with pytest.raises(IndexError, match=msg):
             dfl.iloc[:, 4]
 
+    def test_iloc_non_integer(self):
+
+        # iloc should throw IndexError if non-numeric list is passed
```

Comment:
Can you just add a reference to the GH issue number instead?

Reply:
Fixed